src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -485,6 +485,28 @@ def is_signature_valid(self):
 
         return True
 
+    @property
+    def challenge_password(self):
+        obj = _txt2obj_gc(
+            self._backend, x509.AttributeOID.CHALLENGE_PASSWORD.dotted_string
+        )
+        pos = self._backend._lib.X509_REQ_get_attr_by_OBJ(
+            self._x509_req, obj, -1
+        )
```

Comment:
Sooo, technically attributes are OID + set of values... what happens if there's more than one challenge password?